libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -176,9 +176,6 @@ static int tf_start_inference(void *args)
     if (TF_GetCode(request->status) != TF_OK) {
         av_log(&tf_model->ctx, AV_LOG_ERROR, "%s", TF_Message(request->status));
         tf_free_request(infer_request);
-        if (ff_safe_queue_push_back(tf_model->request_queue, request) < 0) {
-            destroy_request_item(&request);
-        }
```

Comment:
It double calls `destroy_request_item` in line 1152, which would cause segment fault, so I delete the `destroy_request_item` here.